module/cyphersystem.js

```diff
@@ -270,12 +270,12 @@ Hooks.once("dragRuler.ready", (SpeedProvider) => {
       let short = 0;
       let long = 0;
       let veryLong = 0;
-      if (token.scene.data.gridUnits == "m" || token.scene.data.gridUnits == "meter" || token.scene.data.gridUnits == "metre" || token.scene.data.gridUnits == "mètre") {
+      if (token.scene.data.gridUnits == "m" || token.scene.data.gridUnits == "m." || token.scene.data.gridUnits == game.i18n.localize("CYPHERSYSTEM.UnitDistanceMeter")) {
```

Comment:
In the metric system, there are no periods after the unit. Also, please keep “meter“ and “metre” in there for people who use those spellings of the english word. We can’t use both in our translation strings, but people might use them for their maps.